stratum/src/lib.rs

```diff
@@ -83,48 +123,92 @@ pub struct Stratum {
 	workers: Arc<RwLock<HashMap<SocketAddr, String>>>,
 	/// Secret if any
 	secret: Option<H256>,
+	/// Dispatch notify couinter
+	notify_counter: RwLock<u32>,
```

Comment:
there are two operations in a row on that number that cannot be performed atomically (fetch-add + compare-swap)